doc/contributing/documentation_guide.md

```diff
@@ -38,21 +38,38 @@ Use your judgment about what the user needs to know.
 ## General Guidelines
 
 - Keep in mind that the reader may not be fluent in \English.
-- Write short declarative or imperative sentences.
-- Group sentences into (ideally short) paragraphs,
-  each covering a single topic.
-- Organize material with [headers](rdoc-ref:RDoc::Markup@Headers).
-- Refer to authoritative and relevant sources using
-  [links](rdoc-ref:RDoc::Markup@Links).
-- Use simple verb tenses: simple present, simple past, simple future.
-- Use simple sentence structure, not compound or complex structure.
-- Avoid:
-    - Excessive comma-separated phrases;
-      consider a [list](rdoc-ref:RDoc::Markup@Simple+Lists).
+
+- Prefer:
+
+    - Short declarative and imperative sentences.
+    - Short paragraphs, each covering a single topic.
+    - Simple verb tenses: simple present, simple past, simple future.
+    - Simple sentence structure, not compound or complex structure.
+    - Beginning each sentence on a new line
```

Comment:
I'm not against starting sentences on new lines, but I don't think it should be part of the guide. I don't want people to complain if sentences start without a new line. I also don't want pull requests that reformat documentation to add new lines (increasing churn instead of reducing it).

If we have to keep this, I would specify that it should only be done for sentences being added or modified, or those whose lines are affected by a sentence being added or modified.